README.rst

```diff
@@ -11,7 +11,7 @@ The edX Notes API is designed to be compatible with the `Annotator <http://annot
 Getting Started
 ***************
 
-1. Install `ElasticSearch 7.8.0 <https://www.elastic.co/blog/elasticsearch-7-8-0-released>`__.
+1. Install `ElasticSearch 7.13.4 <https://www.elastic.co/downloads/past-releases/elasticsearch-7-13-4>`__.
```

Comment:
Are we upgarding elasticsearch as a part of this PR? can we do that as a separate follow up?  We'll want to announce that and let operators prepare for it.

Reply:
This isn’t an upgrade, just an alignment with the latest supported version per [constraints.txt](https://github.com/openedx/edx-notes-api/blob/2c71edb4aea5751fcdad6784792753c7f88b3199/requirements/constraints.txt#L16) (`elasticsearch>=7.8,<8.0)` and [common_constraints.txt](https://raw.githubusercontent.com/openedx/edx-lint/master/edx_lint/files/common_constraints.txt) (`elasticsearch<7.14.0`). It ensures compatibility, not a major change.